src/app/client/src/app/modules/public/module/explore/components/explore/explore.component.ts

```diff
@@ -131,7 +131,9 @@ export class ExploreComponent implements OnInit, OnDestroy, AfterViewInit {
         if (!this.apiContentList.length) {
           return; // no page section
         }
-        if (this.apiContentList.length >= 2) {
+        if (this.apiContentList.length >= 3) {
```

Comment:
let a = [1, 2, 3];
let b = [1,2];
let c = [];
let arr = a.slice(0, 3);

can u use this logic

Reply:
done